media-video/guvcview/metadata.xml

```diff
@@ -9,5 +9,7 @@
 	</upstream>
 	<use>
 		<flag name="qt5">Build with Qt5 interface instead Gtk+</flag>
+		<flag name="qt6">Build with Qt6 interface</flag>
```

Comment:
if the current (changed from 2.1.0) IUSE logic is kept, then the global description might as well suffice.